doc/indexing.rst

```diff
@@ -57,6 +57,7 @@ DataArray:
 
     Positional indexing deviates from the NumPy when indexing with multiple
     arrays like ``arr[[0, 1], [0, 1]]``, as described in :ref:`indexing details`.
+    Use :py:meth:`~xray.Dataset.isel_points` to achieve this functionality.
```

Comment:
I would reference the section on pointwise indexing in this same document, e.g., by using

    :ref:`pointwise indexing`

as the link and just before the "Pointwise indexing" section below add:

    .. _pointwise indexing: